reference/conanfile/methods.rst

```diff
@@ -353,6 +352,22 @@ recipe has requirements, you can access to your requirements ``user_info`` using
         def build(self):
             self.out.warn(self.deps_user_info["MyLib"].var1)
 
+.. important::
+
+    Both ``env_info`` and ``user_info`` objects store information in a "key <-> value" form and the values are always considered strings.
+    This is done for serialization purposes to *conanbuildinfo.txt* files and to avoid the deserialization of complex structures. However,
+    the information can still be consumed:
```

Comment:
`However...` ~> `It is up to the consumer to convert the string to the expected type:`